From 49e9face8705e03355c5c64ec352eb8bf1b0facb Mon Sep 17 00:00:00 2001
From: Pavan Naregundi <pnaregundi@marvell.com>
Date: Thu, 12 Oct 2023 09:26:10 +0530
Subject: arm64: Select CONFIG_PHY_MVEBU_CP110_COMPHY

This patch selects CONFIG_PHY_MVEBU_CP110_COMPHY with 'y' to fix a
PCI hang.

Change-Id: I175cf64889ed37036bc73d8a06ba8f71e3f23d6d
Signed-off-by: Pavan Naregundi <pnaregundi@marvell.com>
---
 patch/kconfig-inclusions | 1 +
 1 file changed, 1 insertion(+)

diff --git a/patch/kconfig-inclusions b/patch/kconfig-inclusions
index 305a6ed..4686349 100644
--- a/patch/kconfig-inclusions
+++ b/patch/kconfig-inclusions
@@ -107,6 +107,7 @@ CONFIG_MTD_NAND_ECC_SW_BCH=y
 CONFIG_MTD_NAND_MARVELL=m
 CONFIG_SENSORS_EMC2305=m
 CONFIG_KEXEC_FILE=y
+CONFIG_PHY_MVEBU_CP110_COMPHY=y
 
 [marvell-arm64]
 CONFIG_MARVELL_AC5_WATCHDOG=m
-- 
2.25.1

